### ISSUE TITLE
Q: data organization (long term)

### ISSUE THREAD

**Issue by Blind4Basics** [hobovsky/katasniplib#14]
in `content/snippets`, all snippets aren't really discoverable. That's not a problem for now, but once their number grows, it might become very hard to find "the snippet you wanna update or check against a new idea that already exists".

Also, the organization is totally opaque for now. It _seems_ that poly_0001 is storing initial solutions in various languages?

Suggestion:

Use human friendly slugs? possibly adding an automatic ID at the end to enforce uniqueness?

Example: 
```
    snippets/init_sol_0001/python_0001
                          /python_0002
                          /kotlin_0003 (or 001?)
            /random_tests_0002/...
```

**Comment by Blind4Basics** [comment]
if ever the number of scripts becomes large, you'll at least need a way to allow an user raising an issue about one snippet to point exactly the location of your script in your file system. Otherwise you'll get crazy, at some point.

Maybe a right click on the snippet that is showing the path of the snippet? or a tooltip showing it? or both? 

A possibility to copy it to the clipboard would be even better. Context menu:
* option to copy snippet
* separator
* showes the  path 
* option to copy path to clipboard

**Comment by hobovsky** [comment]
Yes, possibility of easily identifying snippets would be good, and, at some point, necessary. For example, the #6 requires the snippets to be identifiable so they can be pointed to. I would also like to make them shareable, so reviewers could paste links to them to support authors ( _"hey, you need to use an assertion with tolerance here, see this link how to use it"_ ). But identification of a snippet is not necessarily related to "browsability" of the library. Both things are good to have, but they don't necessarily have to be solved the same way.

I plan to display some more details of a snippet than just its markdown content. I just did not decide yet what, where, and how.
